src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintExtension.kt

```diff
@@ -18,6 +18,14 @@ open class KtlintExtension {
     var debug = false
 
     /**
+     * Whether or not to allow the build to continue if there are warnings;
+     * defaults to {@code false}, as for any other static code analysis tool.
+     * <p>
+     * Example: {@code ignoreFailures = true}
```

Comment:
In dokka isn't the syntax here supposed to be `[backtick]ignoreFailures = true[backtick]`